mmv1/products/alloydb/User.yaml

```diff
@@ -0,0 +1,103 @@
+  check_response_func_existence: transport_tpg.PollCheckForExistence
+  target_occurrences: 10
+  actions: ['create', 'update', 'delete']
+import_format: ['{{cluster}}/users/{{user_id}}']
+custom_code: !ruby/object:Provider::Terraform::CustomCode
+  custom_import: templates/terraform/custom_import/alloydb_user.go.erb
+autogen_async: true
+skip_sweeper: true
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'alloydb_user'
+    primary_resource_id: 'default'
+      # Fine-grained resource need different autogenerated tests, as
+      # we need to check destroy during a test step where the parent resource
+      # still exists, rather than during CheckDestroy (when read returns
+      # nothing because the parent resource has then also been destroyed)
+    skip_test: true
+    vars:
+      alloydb_user_id: 'me'
+      alloydb_user_password: 'changeme'
+      alloydb_cluster_id: 'alloydb-cluster'
+      alloydb_instance_id: 'alloydb-instance'
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'alloydb_user_iam_user'
+    primary_resource_id: 'default'
+      # Fine-grained resource need different autogenerated tests, as
+      # we need to check destroy during a test step where the parent resource
+      # still exists, rather than during CheckDestroy (when read returns
+      # nothing because the parent resource has then also been destroyed)
+    skip_test: true
+    vars:
+      alloydb_user_id: 'me@example.com'
+      alloydb_cluster_id: 'alloydb-cluster-iam'
+      alloydb_instance_id: 'alloydb-instance-iam'
+parameters:
+  - !ruby/object:Api::Type::String
+    name: 'userId'
+    required: true
+    immutable: true
+    url_param_only: true
+    description: |
+      The ID of the alloydb user.
+  - !ruby/object:Api::Type::ResourceRef
+    name: 'cluster'
+    description: |
+      Identifies the alloydb cluster. Must be in the format
+      'projects/{project}/locations/{location}/clusters/{cluster_id}'
+    required: true
+    immutable: true
+    resource: 'Cluster'
+    imports: 'name'
+    url_param_only: true
+properties:
+  - !ruby/object:Api::Type::String
+    name: 'name'
+    output: true
+    description: |
+      Output only. Name of the resource in the form of projects/{project}/locations/{location}/cluster/{cluster}/users/{user}.
+  - !ruby/object:Api::Type::String
+    name: 'password'
+    description: |
+      Input only. Password for the user. This field is required but shouldn't be set if user_type is set to `ALLOYDB_IAM_USER`
+    sensitive: true
+    ignore_read: true
+  - !ruby/object:Api::Type::Array
+    name: 'databaseRoles'
+    item_type: Api::Type::String
+    description:
+      Optional. List of database roles this user has. The database role strings are subject to the PostgreSQL naming conventions.
+  - !ruby/object:Api::Type::Enum
+    name: 'userType'
+    description: 'Optional. Type of this user.'
+    default_value: :ALLOYDB_BUILT_IN
+    values:
+      - :USER_TYPE_UNSPECIFIED
```

Comment:
Generally we don't need the UNSPECIFIED enum value as it isn't actually settable. So probably remove this.

Reply:
Removed the `UNSPECIFIED` value.